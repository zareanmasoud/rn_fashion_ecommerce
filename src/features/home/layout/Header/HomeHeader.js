import React from 'react';
import MainNavbar from 'features/layout/MainNavbar';
// import styles from './styles';

export default function HomeHeader() {
  return (
    <>
      <MainNavbar noBackButton />
      {/* <SelectMenuByGender /> */}
    </>
  );
}
