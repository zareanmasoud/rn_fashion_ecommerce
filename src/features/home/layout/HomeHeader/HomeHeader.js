import React from 'react';
// eslint-disable-next-line import/no-extraneous-dependencies
import MainNavbar from 'features/layout/MainNavbar';
// import styles from './styles';

const HomeHeader = () => {
  return (
    <>
      <MainNavbar noBackButton />
      {/* <SelectMenuByGender /> */}
    </>
  );
};

export default HomeHeader;
