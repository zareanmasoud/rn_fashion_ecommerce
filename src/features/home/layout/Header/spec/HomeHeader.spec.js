import React from 'react';
import {shallow} from 'enzyme';
import HomeHeader from '../HomeHeader';

describe('HomeHeader Feature', () => {
  describe('Rendering', () => {
    it('should match to snapshot', () => {
      const wrapper = shallow(<HomeHeader />);
      expect(wrapper).toMatchSnapshot();
    });
  });
});
