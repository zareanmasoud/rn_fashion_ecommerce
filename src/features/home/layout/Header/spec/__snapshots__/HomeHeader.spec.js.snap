// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeHeader Feature Rendering should match to snapshot 1`] = `
<Fragment>
  <withNavigation(MainNavbar)
    noBackButton={true}
  />
</Fragment>
`;
